docs/sidebar-examples-configs.ts

```
@@ -71,7 +72,7 @@ export const examples = ([
     config: () => {
       return sidebar.itemsByPaths({
         linkPrefix: '#',
-        sidebarGroupTemplate: (path, _) => {
+        sidebarGroupTemplate: ({ path, depth }) => {
```

ESLint

[error] 'depth' is defined but never used. Allowed unused args must match /^_/u.

(unused-imports/no-unused-vars)